draft-ietf-quic-tls.md

```diff
@@ -1290,40 +1290,38 @@ Endpoints MUST implement and use the replay protections described in {{!TLS13}},
 however it is recognized that these protections are imperfect.  Therefore,
 additional consideration of the risk of replay are needed.
 
-QUIC is not inherently vulnerable to replay attack.  The management of QUIC
-protocol state based on the frame types defined in {{QUIC-TRANSPORT}} is not
-vulnerable to replay.  Processing of QUIC frames is idempotent and cannot result
-in invalid connection states if frames are reordered or lost.  QUIC connections
-do not produce effects that last beyond the lifetime of the connection, except
-for those produced by the application protocol that QUIC serves.
-
-However, this does not count for costs that endpoints might incur as a result of
-accepting 0-RTT.  A server that accepts 0-RTT is exposed to the cost of handling
-a new connection, plus the cost of processing 0-RTT packets.  If replay
-protections are unable to prevent multiple connections from being initiated,
-this could increase these costs because attackers can send copies of 0-RTT
-packets to different server instances, causing the processing to be repeated.
-Servers MUST ensure that they account for any increase in costs before accepting
-connections or 0-RTT.
+QUIC is not vulnerable to replay attack, except via the application protocol
+information it might carry.  The management of QUIC protocol state based on the
+frame types defined in {{QUIC-TRANSPORT}} is not vulnerable to replay.
+Processing of QUIC frames is idempotent and cannot result in invalid connection
+states if frames are replayed, reordered or lost.  QUIC connections do not
+produce effects that last beyond the lifetime of the connection, except for
+those produced by the application protocol that QUIC serves.
+
+Note:
+
+: TLS session tickets and address validation tokens are used to carry QUIC
+  configuration information between connections.  These MUST NOT be used to
+  carry application state.  The potential for reuse of these tokens means that
+  they require stronger protections against replay.
```

Comment:
This contradicts the advice about using session tickets to validate HTTP settings for 0-RTT.